src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
```

Comment:
Add a space after the opening tag `<>` and before the text 'Put the comment here' 🤓